### DISCUSSION TITLE
General info page globally accessible

### DISCUSSION THREAD

**Discussion by wq9578** [filamentphp/filament#10600] [Help] [answered]
How can I implement a publicly accessible info page (for example, for legal requirements such as data privacy information)?
There should be buttons to open it on both public pages and pages requiring user login (panels).

Probably it is best to use the SimplePage class used for the Login page, which is accessible to users not logged in, too.
In panels, a menu item might lead to the info page, and on public pages, render hooks might be used as suggested [here](https://github.com/filamentphp/filament/discussions/7647#discussioncomment-6696533).

**Comment by wq9578** [comment, marked as answer]
Found the answer here: https://github.com/filamentphp/filament/discussions/8843

This should be added to the docs.

**Reply by leandrocfe** [reply]
You can make a PR
